ThirdParty/.clang-tidy

```diff
@@ -0,0 +1,3 @@
+---
+# Disable all checks, one check has to be enabled (hack)
```

Comment:
why the hack?

Reply:
If there is no enabled check clang-tidy (at least in the way it is invoked by CMake) prints out its usage instructions.

Reply:
I did not find another way to specify that this folder should be ignored.

Reply:
I see. I neither do know. ;(
Just add this comment to the "hack", so we are not wondering why...